src/components/FeaturedProjects.js

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable max-len */
+import React from 'react';
+import styled from 'styled-components'
+import { OuterWrapper } from './OuterWrapper';
+import moviesImage from '../images/project-movies.PNG';
+import happyThoughtsImage from '../images/project-happy-thoughts.png';
+import surveyImage from '../images/project-survey.PNG';
+import musicImage from '../images/project-music.PNG';
+
+const OtherProjects = () => {
+	return (
+		<OuterWrapper primary>
+			<FeaturedMoviesStyler className="innerwrapper">
+				<h3>Featured projects</h3>
+				{/* -------------PROJECT MOVIES ------------------*/}
+				<div className="project-flex-wrapper">
+					<div className="project-flex-child1">
+						<a href="https://movie-catalogue-project.netlify.app/" target="_blank" rel="noopener noreferrer">
+							<div className="project-image-group">
+								<img src={moviesImage} alt="movies" />
+								<div className="project-overlay">
+									<h5 className="project-overlay-title" aria-hidden="true">
+											Project movies
+									</h5>
+								</div>
+							</div>
+						</a>
+						<h5 className="project-title" aria-hidden="true">Project movies</h5>
+						<p className="project-paragraph">
+									Imdb alike website where you can see description of movies and their raiting.
+						</p>
+						<div className="tag-wrapper">
+							<p>HTML</p>
+							<p>CSS</p>
+							<p>React</p>
+							<p>API</p>
+							<p>JSX</p>
+							<a href="https://github.com/Petrasoderstrom1612/project-movies" target="_blank" className="repo-tag" rel="noreferrer"><p>GitHub</p>
+							</a>
+						</div>
+					</div>
```

Comment:
It is a great set up, I would recommend that you create a JSON file, called for example data, where you then can fetch data from.